enos/modules/verify_secrets_engines/scripts/kv-pki-verify-certificates.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
+
+set -e
+
+fail() {
+  echo "$1" 1>&2
+  exit 1
+}
+
+[[ -z "$MOUNT" ]] && fail "MOUNT env variable has not been set"
```

Comment:
I think we remove our implicit dependency on certs in the mount. We have a couple options to resolve this:

- Create the names of our root, intermediate, and user cert in the module and pass them as variables to both `create` and `read`. Then, rather than iterating through all the certs with `list` we'll instead explicitly make calls to the cert.
- Create the names and values on `create` and output them in create, then feed that value to read. That's a lot of data so you'll probably want to write it to stdout as JSON or base64 encoded JSON and then feed it to read.

Happy to discuss this more over zoom.

Reply:
updated for read/pki to use environment variables from create/pki